lib/ohai/application.rb

```diff
@@ -99,6 +99,14 @@ def configure_ohai
 
     load_workstation_config
 
+    Ohai::Config.merge!(config)
+
+    if config[:target] || Ohai::Config.target
```

Comment:
shouldn't the merge on line 102 mean that you only have to check the latter one?

Reply:
heh that's all WIP copypasta you shouldn't be reading that too closely, this is not ready for review